### PR TITLE
Adds a cow command

### LINKED ISSUE #208: Random ASCII cow command
https://www.npmjs.com/package/cows

```
        ___________________________
       | (__)  (__)  (__)   (__)  |
       | ( oo  ( oo  ( oo   ( oo  |
_______| /\_|  /\_|  /\_|   /\_|  |________
|                                         |
|   _____                        _____    |
|___|   |________________________|   |____|
    |___|                        |___|
              Cow-pooling
```

### PR DESCRIPTION
**Pull request by pedrofracassi**
Simple stuff. Random cows! `s!cow`

Closes #208

### CODE REVIEW

**Comment by Doges** [comment]
failed

**Comment by bolsomito** [comment]
@pedrofracassi resolve your conflicts blyat

**Comment by LogGame-HU-DEaN** [comment]
Conflicts resolved.

File: package.json [hunk 1]
```diff
@@ -1,73 +1,74 @@
...
+    "ffmpeg-binaries": "^3.2.2-3",
+    "nodemon": "^1.17.5",
+    "standard": "^11.0.1"
+  },
+  "nodemonConfig": {
+    "ignore": [
+      "locales/*"
+    ]
+  }
+}
```
Comment (davipatury) [inline package.json]: ?

Comment (LogGame-HU-DEaN) [reply package.json]: What's wrong with that?

**Pull request merged**